src/main/kotlin/com/getkeepsafe/dexcount/Tasks.kt

```diff
@@ -29,6 +32,36 @@ import java.io.File
  */
 const val MAX_DEX_REFS: Int = 0xFFFF // 65535
 
+@ParallelizableTask
+open class ModernMethodCountTask: DexMethodCountTaskBase() {
+    /**
+     * The output directory of the 'package' task; will contain an
+     * APK or an AAR.
+     */
+    @InputDirectory
+    lateinit var inputDirectory: File
+
+    override val fileToCount: File?
+        get() = inputDirectory.listFiles { _, name ->
+            name?.endsWith(".apk") ?: false
+        }.firstOrNull()
+
+    override val rawInputRepresentation: String
+        get() = "$inputDirectory"
+}
+
+@ParallelizableTask
+open class LegacyMethodCountTask: DexMethodCountTaskBase() {
+
+    lateinit var variantOutput: BaseVariantOutput
+
+    override val fileToCount: File?
+        get() = variantOutput.outputFile
+
+    override val rawInputRepresentation: String
+        get() = "variantOutput={name=${variantOutput.name} outputFile=${variantOutput.outputFile}}"
```

Comment:
Not sure what you mean - are you suggesting changing `outputFile=${variantOutput.outputFile}` to `outputFile = ${variantOutput.outputFile}`?